inc/Services/Assets.php

```diff
@@ -126,32 +131,27 @@ public function get_min_file( string $type ): string {
 			return '';
 		}
 
-		if ( ! file_exists( \get_theme_file_path( '/dist/assets.json' ) ) ) {
-			return '';
-		}
+		$assets = $this->get_assets_json_index_file();
 
-		$json   = file_get_contents( \get_theme_file_path( '/dist/assets.json' ) ); //phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
-		$assets = json_decode( $json, true );
-
-		if ( empty( $assets ) || JSON_ERROR_NONE !== json_last_error() ) {
+		if ( empty( $assets ) ) {
 			return '';
 		}
 
 		switch ( $type ) {
 			case 'css':
-				$file = $assets['app.css'];
+				$file = $assets['app.css'] ?? '';
 				break;
 			case 'editor.css':
-				$file = $assets['editor.css'];
+				$file = $assets['editor.css'] ?? '';
 				break;
 			case 'login':
-				$file = $assets['login.css'];
+				$file = $assets['login.css'] ?? '';
 				break;
 			case 'editor.js':
-				$file = $assets['editor.js'];
+				$file = $assets['editor.js'] ?? '';
 				break;
 			case 'js':
-				$file = $assets['app.js'];
+				$file = $assets['app.js'] ?? '';
 				break;
 			default:
 				$file = null;
```

Comment:
Plutôt que d'avoir le code pour le custom type après, je l'aurais déplacé ici :
```suggestion
				$file = $assets[ $type ] ?? '';
```

Et supprimer la partie après :
```php
// Custom type
if ( ! empty( $assets[ $type ] ) ) {
	$file = $assets[ $type ];
}
```